docs/release/role-handbooks/ci-signal/README.md

```diff
@@ -45,7 +45,7 @@ While we add test coverage for the new release branch we will also drop the test
  ```sh
   TEST_INFRA_DIR=../../k8s.io/test-infra make generate-test-infra-prowjobs
   ```
-6. Verify the jobs and dashboards a day later by taking a look at: `https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-1.8`
+6. Verify the jobs and dashboards a day later by taking a look at: `https://testgrid.k8s.io/cluster-api-core-1.8`
```

Comment:
This link will break pretty soon. We should probably link to main instead?

Reply:
maybe?  just in keeping with this statement https://github.com/kubernetes-sigs/cluster-api/blob/82678056845a1368406261158d9efd5d7fb6b677/docs/release/role-handbooks/ci-signal/README.md?plain=1#L34

this whole doc could probably use a thorough update though

Reply:
Yeah. It's just that I know this will be a broken link in 2 weeks. Maybe we can just link the main page and say click on a button?

Reply:
main page == https://testgrid.k8s.io/cluster-api-core